src/XMakeBuildEngine/Utilities/Utilities.cs

```diff
@@ -459,7 +459,7 @@ internal static PropertyDictionary<ProjectPropertyInstance> GetEnvironmentProper
             // environment or as a global property.  
 
 #if !FEATURE_INSTALLED_MSBUILD
```

Comment:
When you're in town next week, I'd love to discuss this.  I see all these "feature flags" and I've heard of the concept but am not entirely sold.  But I'm assuming their useful and I just don't get it?